File: beginner/java/2779.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int N = scanner.nextInt();
        int M = scanner.nextInt();

        int[] album = new int[N + 1];

        for (int i = 1; i <= N; i++) {
            album[i] = 0;
        }

        for (int i = 0; i < M; i++) {
            int figurinha = scanner.nextInt();
            album[figurinha] = 1;
        }

        int faltam = 0;
        for (int i = 1; i <= N; i++) {
            if (album[i] == 0) {
                faltam++; 
            }
        }

        System.out.println(faltam);

        scanner.close();
    }
}

File: beginner/c/2779.c
#include <stdio.h>

int main() {
    int N, M;
    scanf("%d", &N);
    scanf("%d", &M);

    int album[N + 1];
    for (int i = 1; i <= N; i++) {
        album[i] = 0; 
    }

    for (int i = 0; i < M; i++) {
        int figurinha;
        scanf("%d", &figurinha);
        album[figurinha] = 1;
    }

    int faltam = 0;
    for (int i = 1; i <= N; i++) {
        if (album[i] == 0) {
            faltam++; 
        }
    }

    printf("%d\n", faltam);

    return 0;
}
